web/src/routes/Alerts.tsx

```diff
@@ -0,0 +1,22 @@
+import { Alert, AlertActionCloseButton, AlertGroup } from "@patternfly/react-core";
+import { useRecoilState } from "recoil";
+import { alertState } from "src/atoms/AlertState";
+
+
+export default function Alerts(){
+    const [alerts, setAlerts] = useRecoilState(alertState);
+    return (
+    <AlertGroup isToast isLiveRegion>
+        {alerts.map(alert=><Alert
```

Comment:
nit: doesn't seem to be formatted. It would be a good idea to add a format check for .tsx files in our CI

Reply:
agreed